crates/unftp-sbe-gcs/src/lib.rs

```diff
@@ -273,6 +271,7 @@ impl<User: UserDetail> StorageBackend<User> for CloudStorage {
         let request: Request<Body> = Request::builder()
             .uri(uri)
             .header(header::AUTHORIZATION, format!("Bearer {}", token))
+            .header("Range", format!("bytes={}-", start_pos))
```

Comment:
Drive-by review: it looks like the Range header [exists as a constant in hyper](https://docs.rs/hyper/latest/hyper/header/constant.RANGE.html), so let's use it

```suggestion
            .header(header::RANGE, format!("bytes={}-", start_pos))
```

Reply:
a nice one

Reply:
fixed